bridge/setu/util/common.go

```diff
@@ -152,7 +153,8 @@ func IsInProposerList(cliCtx cliContext.CLIContext, count uint64) (bool, error)
 
 // CalculateTaskDelay calculates delay required for current validator to propose the tx
 // It solves for multiple validators sending same transaction.
-func CalculateTaskDelay(cliCtx cliContext.CLIContext) (bool, time.Duration) {
+func CalculateTaskDelay(cliCtx cliContext.CLIContext, event interface{}) (bool, time.Duration) {
```

Comment:
get time for unconfirmed txns as well

Reply:
@0xKrishna do you think adding mempool check will increase the delay time by a lot?

Reply:
Done, thank you

Reply:
It should only come into the picture when the txn is already in mempool, And if it's in mempool that means some bridge has already processed it and should get included in a block soon, But yeah we should add logs here as well to confirm if this is causing unnecessary delay.